apps/src/p5lab/spritelab/SpriteLab.js

```diff
@@ -59,3 +60,23 @@ SpriteLab.prototype.onPause = function(isPaused) {
     coreLibrary.startPause(current);
   }
 };
+
+SpriteLab.prototype.setupReduxSubscribers = function(store) {
+  P5Lab.prototype.setupReduxSubscribers.call(this, store);
+  var state = {};
```

Comment:
2 questions:
1. why assign `state` here instead of on line 69 where it gets re-assigned?
2. is there a reason we're using `var` instead of `let` or `const` throughout this function?

Reply:
1. We use the value on line 68 (before re-assigning) so it needs to be initialized outside the closure
2. Done :)